### PR TITLE
Notif 285 - List event types in table format

### PR DESCRIPTION
**Pull request by aferd**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (josejulio) [inline yarn.lock:1]: I think this file should be deleted, doesn't seem to be part of the webapp itself.

File: node_modules/.yarn-integrity [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (josejulio) [inline node_modules/.yarn-integrity:1]: I think this file should be deleted, doesn't seem to be part of the webapp itself.

**Comment by aferd** [comment]
Screen Shot 2021-08-20 at 1 52 33 PM

File: backend/src/main/webapp/src/pages/ApplicationPage.tsx [hunk 3]
```diff
@@ -25,10 +30,34 @@ export const ApplicationPage: React.FunctionComponent = () => {
...
+        <React.Fragment>
+            <PageSection>
+                <Title headingLevel="h1"><Breadcrumb>
+                    <BreadcrumbItem to='#'> Red Hat Enterprise Linux </BreadcrumbItem>
+                    <BreadcrumbItem to='#'isActive> { (applicationTypesQuery.loading || applicationTypesQuery.payload?.status !== 200) ?
+                        <Spinner /> : applicationTypesQuery.payload.value.displayName } </BreadcrumbItem>
+                </Breadcrumb></Title>
+            </PageSection>
+            <TableComposable
+                aria-label="Simple table"
```
Comment (josejulio) [inline backend/src/main/webapp/src/pages/ApplicationPage.tsx:42]:
```suggestion
                aria-label="Event types table"
```

**Review by josejulio — APPROVED** [review]

File: backend/src/main/webapp/src/pages/__tests__/ApplicationPage.test.tsx [hunk 4]
```diff
@@ -0,0 +1,8 @@
+// import * as React from 'react'
+// import {render, screen} from '@testing-library/react'
+// import { ApplicationPage } from '../ApplicationPage'
+// import { Table, TableComposable } from '@patternfly/react-table'
+
+// describe ('src/pages/ApplicationPage', () => {
+
+// })
```
Comment (josejulio) [inline backend/src/main/webapp/src/pages/__tests__/ApplicationPage.test.tsx:8]: Tests are in a pending state for the whole UI. Lets work on this in a separate PR.

**Pull request merged**